bbb-graphql-middleware/config/config.yml

```diff
@@ -5,7 +5,7 @@ server:
   max_connections: 500
   max_connections_per_session_token: 3
   # Rate limit: maximum number of new connections/clients allowed per second.
-  max_connections_per_second: 100
+  max_connections_per_second: 50
```

Comment:
While determining the best default value for this configuration, I tested several different options, and at the time, `100` produced the best results. You can find more details here: https://github.com/bigbluebutton/bigbluebutton/pull/20754.

Are you certain this change will actually improve the quality?

Reply:
Let's keep this change for another pull request. Perhaps @gustavotrott we need to do some testing and come up with some guide like:
0 - 400 users -- Default: single hasura, no real point in launching second (useful for smaller servers or if you loadbalance it so that you rarely have over 300 users per server)
400 - 800 users -- run two instances of hasura if you expect average of 300+ users but less than 600
800+ run 4 instances of hasura but keep in mind that something's going to give out, and it won't be hasura this time...

Not sure if there would be clear dependency between # hasuras and the max_connections_per_second...
How much sense does it make to have 2 hasuras with lower number of connections per second vs 1 hasura with higher?